device_aarch64.go

```diff
@@ -0,0 +1,5 @@
+// +build aarch64 arm64
+
+package qcli
+
+const TPMTISDeviceAarch64 DeviceDriver = "tpm-tis-device"
```

Comment:
```suggestion
const TPMTISDevice DeviceDriver = "tpm-tis-device"
```

Since this is only compiled in for aarch64, no need to include the arch name in the string.

Reply:
Would this conflict with the `TPMTISDevice` in the regular `device.go` file (if both are built)

Reply:
> Would this conflict with the `TPMTISDevice` in the regular `device.go` file (if both are built)

Correct, but since you're marking this file with // build +aarch64 ; it will only get included in the arm build.

Reply:
> > Would this conflict with the `TPMTISDevice` in the regular `device.go` file (if both are built)
>
> Correct, but since you're marking this file with // build +aarch64 ; it will only get included in the arm build.

Ah, then I guess we'd also need to push the TPMTisDevice into a device_amd64.go.

Here, I think let's leave this TPMTISDeviceAarch64 out; and instead tpm.go, we can use a switch and goarch

```golang
func (tpm TPMDevice) deviceName() string {
    switch tpm.Driver {
    case TPMTISDevice:
          if runtime.GOARCH == "aarch64" || runtime.GOARCH == "arm64" {
              return string(tpm.Driver + "-device")
          }
    }
    return string(tpm.Driver)
}
```

Reply:
Yep this makes sense, will include